platforms/jvm/scala/src/main/java/org/gradle/api/tasks/scala/ScalaForkOptions.java

```diff
@@ -34,10 +34,16 @@
 
 import org.gradle.api.tasks.compile.ProviderAwareCompilerDaemonForkOptions;
 
+import java.util.Collections;
+
 /**
  * Fork options for Scala compilation. Only take effect if {@code BaseScalaCompileOptions.fork}
  * is {@code true}.
  */
 public abstract class ScalaForkOptions extends ProviderAwareCompilerDaemonForkOptions {
     private static final long serialVersionUID = 0;
+
+    public ScalaForkOptions() {
+        setJvmArgs(Collections.singletonList("-Dxsbt.skip.cp.lookup=true"));
```

Comment:
❌ After looking some more, it might be better to add this option explicitly at a lower level. `DaemonScalaCompiler.toForkOptions` seems like the right place.

Since Gradle depends on this option for correctness, it should be set at a level where the user cannot accidentally unset it. Unless having it twice is not a problem, we should check for its presence before adding it.

Reply:
Thanks!

Since it's a system prop, setting it twice is fine. I've moved it to daemon